packages/remark-codesandbox/src/index.js

```diff
@@ -101,10 +101,19 @@ function codesandbox(options = {}) {
         );
       }
 
+      const overrideEntry =
+        query.get('overrideEntry') !== 'false' ||
+        options.overrideEntry ||
+        true;
+
+      // Remove any options that are only for the plugin and not relevant to CodeSandbox
+      const pluginOnlyQueryParams = ['overrideEntry'];
```

Comment:
Nitpick/Coding Style: Prefer to name this as SNAKE_CASE (all caps) and move it to the top of the file.

Reply:
Sounds good! I was looking for something similar actually in the file, but when I didn't find it, I decided to not introduce something new.